src/modules/vendor/components/companies/ProfileInfo.vue

```diff
@@ -238,13 +239,18 @@ export default {
         this.users = [];
       }
     },
+    formatUserPayload (user) {
+      const userPayload = pickBy(user);
+      userPayload.company = this.company._id;
+      return userPayload;
```

Comment:
`return { ...pickBy(user), company: this.company._id };`

Reply:
Figure-toi que j'y ai pensé mais je me suis dit que le spread rajoutait une opération en plus. ^^